translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+        &lt;ol&gt;
+          &lt;li&gt;Click on one of your pieces.&lt;/li&gt;
+          &lt;li&gt;Click on one of the triangular space highlighted in yellow.&lt;/li&gt;
+        &lt;/ol&gt;
+        You can pass through the opponent&apos;s pieces.
+        You&apos;re playing first, hence you&apos;re playing Dark.&lt;br/&gt;&lt;br/&gt;
+        Perform any move.</source>
+        <target>Pour effectuer un déplacement, il faut :
+        &lt;ol&gt;;
+            &lt;li&gt;Cliquer sur l&apos;une de vos pièces.&lt;/li&gt;
+            &lt;li&gt;Cliquer sur l&apos;une des cases triangulaires encadrées en jaune.&lt;/li&gt;
+        &lt;/ol&gt;
+        Vous pouvez passer à travers les pièces adverses.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez en premier, vous jouez donc Foncé, faites n&apos;importe quel déplacement.&lt;br/&gt;
+        Note : peut importe ce que vous faites, aucunes de vos pièces ne peuvent être capturée pendant votre tour.</target>
+      </trans-unit>
+      <trans-unit id="3313068005460528101" datatype="html">
+        <source>Congratulations! Let&apos;s see captures now.</source>
+        <target>Bravo, voyons ensuite les captures.</target>
+      </trans-unit>
+      <trans-unit id="7869356423919656180" datatype="html">
+        <source>Capture</source>
+        <target>Capture</target>
+      </trans-unit>
+      <trans-unit id="4864789526486078372" datatype="html">
+        <source>Every piece has three neighboring triangular spaces (2 on the sides).
+        When all neighboring spaces except one are occupied, and one opponent moves to that last free space, your piece is captured!
+        However, it is possible to place a piece between 3 of the opponent&apos;s pieces (or 2 on the side) without being captured.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Light, make a capture.</source>
+        <target>Chaque pièce a trois cases triangulaires voisines (2 sur les bords).
+        Quand toutes les cases voisines sauf une sont occupées, et qu&apos;une pièce de l'adversaire vient se déplacer sur cette dernière case libre, votre pièce est capturée !
+        Cependant, il est possible pour un joueur de se placer entre 3 pièces adverses (ou 2 contre un bord) sans être capturé.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez Clair, effectuez une capture</target>
+      </trans-unit>
+      <trans-unit id="1766583918856668821" datatype="html">
+        <source>Failed, you have not captured any piece.</source>
+        <target>Raté, vous n&apos;avez pas capturé de pièce !</target>
+      </trans-unit>
+      <trans-unit id="8225905705628695723" datatype="html">
+        <source>Gain a tile</source>
+        <target>Gagner une tuile</target>
+      </trans-unit>
+      <trans-unit id="362405963474133663" datatype="html">
+        <source>When a tile is empty, it can become removable from the board.
+        For it to be removable, at least three of its sides must be free, and these should be consecutive sides.
```

Comment:
these should be neighbors sides. ?
J'ai peur que consécutif pour les prophanes, ça pique!

Reply:
changé en "neighboring sides"

Reply:
No you didn't ! 
Et je regarde bien "changes from all commits"! :D

Reply:
maintenant c'bon, oops!